interactions/client/get.py

```diff
@@ -0,0 +1,394 @@
+from asyncio import sleep
+from enum import Enum
+from inspect import isawaitable, isfunction
+from logging import getLogger
+from typing import Coroutine, Iterable, List, Optional, Type, TypeVar, Union, get_args
+
+try:
+    from typing import _GenericAlias
+
+except ImportError:
+    from typing import _BaseGenericAlias as _GenericAlias
+
+from sys import version_info
+
+from ..api.error import LibraryException
+from ..api.http.client import HTTPClient
+from ..api.models.channel import Channel
+from ..api.models.guild import Guild
+from ..api.models.member import Member
+from ..api.models.message import Emoji, Message, Sticker
+from ..api.models.misc import Snowflake
+from ..api.models.role import Role
+from ..api.models.user import User
+from ..api.models.webhook import Webhook
+from .bot import Client
+
+log = getLogger("get")
+
+_A = TypeVar("_A", Channel, Guild, Webhook, User, Sticker, Message, Emoji, Role, Message)
```

Comment:
```suggestion
_T = TypeVar("_T")
```

This is a **huge** change in typing annotation behaviour, but I'm suggesting this as restricting the types will only serve be problematic later on in time when the API releases even more objects that we can either pull from cache or make http calls for. For example, I can't get information on:

- Application commands
- Components
- Webhooks

Reply:
I have another definition of _T in the pyi already tho